projects/plugins/jetpack/CHANGELOG.md

```diff
@@ -4,35 +4,27 @@
 
 ## 11.2-a.3 - 2022-07-12
 ### Enhancements
-- Add new `jetpack_podcast_pre_fetch` and `jetpack_podcast_post_fetch` actions to make it possible for users to set up code that runs for podcast fetches. [#25046]
-- Jetpack: create and update Icon dependency for VPBlock v6 [#24974]
-- Jetpack: improve initial rendering of the VPBlock [#24891]
-- Jetpack: improve VideoPress Block placeholders [#25058]
-- Jetpack: replace local Loading by Spinner in VideoPress block v6 [#24975]
-- Jetpack: stop re-attempting to get the video preview when it achieves the limit [#24897]
-- Jetpack: update the VideoPressIcon path for v6 [#24981]
-- VideoPress: Implement upload progress bar [#24953]
-- VideoPress: Improve upload error layout [#24963]
-
-### Improved compatibility
-- add videopress package as dependency [#25045]
+- Podcast Player: add new actions to make it possible for users to set up code that runs for podcast fetches. [#25046]
+- VideoPress: add upload progress bar. [#24953]
+- VideoPress: improve the UX when rendering the block. [#24891]
+- VideoPress: improve upload error layout. [#24963]
+- VideoPress: limit preview attempts. [#24897]
```

Comment:
If these are all for the new VP block, perhaps these could be merged and mention the block is in beta?

Reply:
Oh they are - the functionality doesn't exist with regular testing on WoA actually so I wonder if they should be removed entirely too...

Reply:
I think merging to one line would be good since with the beta plugin users can use the Atomic releases.

Reply:
Agree, added a new commit: 85f905f11dc6939c3a365b0385c3281b400344d9